website/docs/working-with-containers/registries/_insecure-registry-linux.md

```diff
@@ -0,0 +1,37 @@
+#### Prerequisites
+
+- Podman is installed.
+
+#### Procedure
+
+1. Add your insecure registry within **<icon icon="fa-solid fa-cog" size="lg" /> Settings > Registries**.
+
+   ![Adding a custom registry](img/adding-a-custom-registry.png)
+
+1. Click "Yes" to the insecure registry warning.
+
+   ![Podman Desktop Registry Warning](img/registry-warning-insecure.png)
+
+1. Open `registries.conf`.
+
+   ```shell-session
+   $ sudo vi /etc/containers/registries.conf
```

Comment:
Isn't it in openshift docs / standards that sudo commands use `#` / comment out feature so it's not accidently copied-and-pasted? unsure if we should apply this here / other parts of PD documentation.